### ISSUE TITLE
Buttons and other text missing from Pattypan that get fixed by making fullscreen or reducing window size

### ISSUE THREAD

**Issue by mrjohnc** [yarl/pattypan#137]
I'm getting some very strange behaviour in PattyPan where buttons, text or everything is missing from the page which can be fixed if I make it fullscreen, or if it is in fullscreen making it smaller. See screenshots for examples.
![Screenshot (108)](https://user-images.githubusercontent.com/7293392/130231366-6b169048-7bab-4323-82b4-e51a5c76c34b.png)
![Screenshot (109)](https://user-images.githubusercontent.com/7293392/130231372-edc02a80-7b54-4d55-8fc0-e771babf756e.png)
![Screenshot (110)](https://user-images.githubusercontent.com/7293392/130235198-9eab371c-70c4-4b3d-b7c0-6b7af37c18c5.png)
![Screenshot (111)](https://user-images.githubusercontent.com/7293392/130235227-0b8a34d5-f153-4407-9a3d-6ca8b8398b8f.png)

**Comment by Abbe98** [comment]
That is odd, could you share the error log(should be a link to the file on the first screen)? Alternatively starting it from the command line `java --jar pattypan.jar` and share the output.

**Comment by mrjohnc** [comment]
Hmmm I get this

Unrecognized option: --jar
Error: Could not create the Java Virtual Machine.
Error: A fatal exception has occurred. Program will exit.

Is it because I'm on Windows?

**Comment by mrjohnc** [comment]
Jul 06, 2021 6:21:41 PM pattypan.Settings readProperties
INFO: Settings file not found, use default
Jul 06, 2021 6:21:41 PM pattypan.Main main
INFO: Wiki set as: commons.wikimedia.org
Protocol set as: https://
Script path set as: /w
Jul 06, 2021 6:21:41 PM pattypan.Main main
INFO: Operating System: Windows 10
Pattypan Version: 20.04
Jul 06, 2021 6:23:00 PM pattypan.Main main
INFO: Wiki set as: commons.wikimedia.org
Protocol set as: https://
Script path set as: /w
Jul 06, 2021 6:23:00 PM pattypan.Main main
INFO: Operating System: Windows 10
Pattypan Version: 20.04
Aug 06, 2021 8:38:55 PM pattypan.Main main
INFO: Wiki set as: commons.wikimedia.org
Protocol set as: https://
Script path set as: /w
Aug 06, 2021 8:38:55 PM pattypan.Main main
INFO: Operating System: Windows 10
Pattypan Version: 20.04
Aug 06, 2021 8:41:47 PM pattypan.panes.CreateFilePane getExifDate
INFO: Exif error for 20170209_un_communicating_sust_lifestyles_fullreport_lores_2016.pdf: File format could not be determined
Aug 06, 2021 8:41:47 PM pattypan.panes.CreateFilePane getExifDate
INFO: Exif error for Gmail - Your Belkin.co.uk Order has been Confirmed - #EC08157751.pdf: File format could not be determined
Aug 06, 2021 8:41:47 PM pattypan.panes.CreateFilePane getExifDate
INFO: Exif error for IC160UK_Series_IB_E_MP_200907.pdf: File format could not be determined
Aug 06, 2021 9:13:26 PM pattypan.Main main
INFO: Wiki set as: commons.wikimedia.org
Protocol set as: https://
Script path set as: /w
Aug 06, 2021 9:13:26 PM pattypan.Main main
INFO: Operating System: Windows 10
Pattypan Version: 20.04
Aug 06, 2021 9:20:00 PM pattypan.Main main
INFO: Wiki set as: commons.wikimedia.org
Protocol set as: https://
Script path set as: /w
Aug 06, 2021 9:20:00 PM pattypan.Main main
INFO: Operating System: Windows 10
Pattypan Version: 20.04
Aug 20, 2021 9:38:36 AM pattypan.Main main
INFO: Wiki set as: commons.wikimedia.org
Protocol set as: https://
Script path set as: /w
Aug 20, 2021 9:38:36 AM pattypan.Main main
INFO: Operating System: Windows 10
Pattypan Version: 20.04
Aug 20, 2021 12:26:30 PM pattypan.Main main
INFO: Wiki set as: commons.wikimedia.org
Protocol set as: https://
Script path set as: /w
Aug 20, 2021 12:26:30 PM pattypan.Main main
INFO: Operating System: Windows 10
Pattypan Version: 20.04
Aug 20, 2021 12:41:37 PM pattypan.Main main
INFO: Wiki set as: commons.wikimedia.org
Protocol set as: https://
Script path set as: /w
Aug 20, 2021 12:41:37 PM pattypan.Main main
INFO: Operating System: Windows 10
Pattypan Version: 20.04
Aug 20, 2021 12:47:55 PM pattypan.Main main
INFO: Wiki set as: commons.wikimedia.org
Protocol set as: https://
Script path set as: /w
Aug 20, 2021 12:47:55 PM pattypan.Main main
INFO: Operating System: Windows 10
Pattypan Version: 20.04
Aug 20, 2021 12:51:53 PM pattypan.Main main
INFO: Wiki set as: commons.wikimedia.org
Protocol set as: https://
Script path set as: /w
Aug 20, 2021 12:51:53 PM pattypan.Main main
INFO: Operating System: Windows 10
Pattypan Version: 20.04
Aug 20, 2021 1:00:22 PM pattypan.Main main
INFO: Wiki set as: commons.wikimedia.org
Protocol set as: https://
Script path set as: /w
Aug 20, 2021 1:00:22 PM pattypan.Main main
INFO: Operating System: Windows 10
Pattypan Version: 20.04
Aug 20, 2021 1:11:45 PM pattypan.Main main
INFO: Wiki set as: commons.wikimedia.org
Protocol set as: https://
Script path set as: /w
Aug 20, 2021 1:11:45 PM pattypan.Main main
INFO: Operating System: Windows 10
Pattypan Version: 20.04
Aug 20, 2021 1:45:21 PM pattypan.Main main
INFO: Wiki set as: commons.wikimedia.org
Protocol set as: https://
Script path set as: /w
Aug 20, 2021 1:45:21 PM pattypan.Main main
INFO: Operating System: Windows 10
Pattypan Version: 20.04
Aug 20, 2021 6:31:46 PM pattypan.Main main
INFO: Wiki set as: commons.wikimedia.org
Protocol set as: https://
Script path set as: /w
Aug 20, 2021 6:31:46 PM pattypan.Main main
INFO: Operating System: Windows 10
Pattypan Version: 20.04

**Comment by Abbe98** [comment]
@mrjohnc Do you happen to have a 4K screen or custom screen zoom?

**Comment by mrjohnc** [comment]
Yes I think its 4K, display settings says it 3840x2400, no screen zoom or anythig like that. Just standard Windows laptop I bought recently
